docs/backends/linux.rst

```diff
@@ -41,6 +41,12 @@ individual Bleak objects should not be shared between event loops. Otherwise,
 RuntimeErrors similar to ``[...] got Future <Future pending> attached to a
 different loop`` will be thrown.
 
+User Namespaces
```

Comment:
We have another similar request for anonymous d-bus authentication at https://github.com/hbldh/bleak/pull/796 that was never resolved. So perhaps it makes sense to make this section about "D-Bus authentication" in general instead of just user namespaces. Then we can add to it in the other PR.

Also, to start a pattern, perhaps we could make this environment variable `BLEAK_DBUS_AUTH_UUID` and use `BLEAK_DBUS_AUTH_ANON` in the other PR.

Reply:
Sounds reasonable, will do!

To clarify, do you mean `BLEAK_DBUS_AUTH_UID` (vs `UUID`)?

Reply:
I meant that we should replace `BLEAK_DBUS_UUID` with `BLEAK_DBUS_AUTH_UUID` in this PR.

Reply:
Yup, I get that - but do you explicitly want it to be [**U** **U** **I** **D**](https://en.wikipedia.org/wiki/Universally_unique_identifier)?

Reply:
Ha, oops, didn't even notice that. Should be UID, of course!